Octokit.Tests.Integration/Clients/CommitStatusClientTests.cs

```diff
@@ -28,6 +28,27 @@ public class TheGetAllMethod
         }
     }
 
+    public class TheGetCombinedMethod
+    {
+        [IntegrationTest]
+        public async Task CanRetrieveCombinedStatus()
```

Comment:
Add in a test for this commit - it has multiple statuses, so you can test that we're handling collections correctly: https://github.com/libgit2/libgit2sharp/commit/f54529997b6ad841be524654d9e9074ab8e7d41d